roles/arteria-sequencing-report-ws/tasks/install.yml

```diff
@@ -29,9 +29,9 @@
 - name: download seqreports singularity images
   shell: |
     module load bioinfo-tools Nextflow/21.04
-    nextflow -log /scratch run {{ arteria_service_src_path }}/seqreportsmain.nf -profile test,singularity
+    nextflow -log /scratch run {{ arteria_service_src_path }}/seqreports/main.nf -profile test,singularity
```

Comment:
Could you replace scratch with the ENV parameter here as well?

Reply:
Done!